samples/sensor/magn_trig/sample.yaml

```diff
@@ -6,6 +6,7 @@ tests:
     tags: sensors
     filter: dt_alias_exists("magn0")
     harness_config:
+      fixture: fixture_sensor_accel_int
```

Comment:
sensor_magn_int perhaps was wanted here?

Reply:
this could be better, as NXP use the same device for accel and magn,